Mobile Buy SDK/Mobile Buy SDK Tests/BUYClient+StorefrontTests.m

```diff
@@ -285,7 +285,7 @@ - (void)testProductsInCollection
 	
 	XCTestExpectation *expectation = [self expectationWithDescription:NSStringFromSelector(_cmd)];
 
-	[self.client getProductsPage:1 inCollection:self.collection.identifier completion:^(NSArray *products, NSUInteger page, BOOL reachedEnd, NSError *error) {
+	[self.client getProductsPage:1 inCollection:self.collection.identifier withTags:@[@"cotton", @"summer"] sortOrder:BUYCollectionSortCollectionDefault completion:^(NSArray *products, NSUInteger page, BOOL reachedEnd, NSError *error) {
```

Comment:
This will need to be updated at some point, as a new property was added on the test client for tags